agents/arduino/arduino_agent.py

```diff
@@ -58,6 +64,9 @@ def init_arduino(self):
 
 
     def start_acq(self, session, params):
+        '''
+        Starts acquiring data from an Arduino.
```

Comment:
We've been documenting arugments in `params` as the arguments to the function. See the 240 Agent for examples.